Gemfile

```diff
@@ -3,11 +3,3 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
-
-group :development do
-  gem 'rspec'
```

Comment:
The rspec has to stay with the cucumber or the tests will not run:
https://travis-ci.org/dmcnulla/ntp/builds/121730524

I can move the other to Gemfile.

Thanks,

Dave McNulla


On Fri, Apr 8, 2016 at 10:52 PM, Павел Скрылёв (Malo Skrylevo) <
notifications@github.com> wrote:

> In Gemfile <https://github.com/dmcnulla/ntp/pull/1#discussion_r59110909>:
>
> > @@ -3,11 +3,3 @@ source 'https://rubygems.org'
> >  gemspec
> >
> >  gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
> > -
> > -group :development do
> > -  gem 'rspec'
>
> well the I believe pry, rubocop, and rspec can be moved back to Gemfile,
> in order to not waste gemspec =)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dmcnulla/ntp/pull/1/files/24a259dc48a3962964a53e46b3778e6441c8ae2c..1ec80f8d6a1bf2887d99a8bd6f079fbd1167a85d#r59110909>
>